documentation/tutorial/08-deploy-worker.md

```diff
@@ -0,0 +1,99 @@
+
+```bash
+juju add-model temporal-worker
+juju deploy temporal-worker-k8s
+```
+
+Wait until the application is ready - when it is ready, `juju status` will show:
+
+```
+Model            Controller           Cloud/Region        Version  SLA          Timestamp
+temporal-worker  temporal-controller  microk8s/localhost  3.1.5    unsupported  13:21:49+03:00
+
+App                  Version  Status   Scale  Charm                Channel  Rev  Address         Exposed  Message
+temporal-worker-k8s           waiting      1  temporal-worker-k8s  stable     5  10.152.183.187  no       installing agent
+
+Unit                    Workload  Agent  Address      Ports  Message
+temporal-worker-k8s/0*  blocked   idle   10.1.232.75         Invalid config: wheel-file-name missing
+```
+
+## Configure Worker
+
+In order to test our Temporal worker, we will use a
+[sample resource](https://github.com/canonical/temporal-worker-k8s-operator/tree/main/resource_sample)
+which contains one workflow and one activity. Once downloaded, you can run the
+following command to build it and attach it to the worker charm:
+
+```bash
+poetry build -f wheel # Take note of the generated file's name
+```
+
+We will then configure our worker to the Temporal server deployed in the
+previous steps. Create a file `config.yaml` with the following content:
+
+```yaml
+temporal-worker-k8s:
+  host: "10.1.232.64:7233" # Temporal Server unit IP address
+  queue: "test-queue"
+  namespace: "default"
+  workflows-file-name: "python_samples-1.1.0-py3-none-any.whl"
+  # To support all defined workflows and activities, use the 'all' keyword
+  supported-workflows: "all"
+  supported-activities: "all"
+```
+
+Run the following command to configure your charm:
+
+```bash
+juju config temporal-worker-k8s --file=path/to/config.yaml
+```
+
+And then run the following command to attach the workflows file to the worker:
+
+```bash
+juju attach-resource temporal-worker-k8s workflows-file=./dist/python_samples-1.1.0-py3-none-any.whl
+```
+
+Wait until the application is ready - when it is ready, `juju status` will show:
+
+```
+Model            Controller           Cloud/Region        Version  SLA          Timestamp
+temporal-worker  temporal-controller  microk8s/localhost  3.1.5    unsupported  13:45:16+03:00
+
+App                  Version  Status  Scale  Charm                Channel  Rev  Address         Exposed  Message
+temporal-worker-k8s           active      1  temporal-worker-k8s  stable     5  10.152.183.187  no       worker listening to namespace 'default' on queue 'test-queue'
+
+Unit                    Workload  Agent  Address      Ports  Message
+temporal-worker-k8s/0*  active    idle   10.1.232.78         worker listening to namespace 'default' on queue 'test-queue'
+```
+
+To further verify that the worker is functioning correctly, observe the output
+of the following command:
+
+```bash
+juju ssh --container temporal-worker temporal-worker-k8s/0 /charm/bin/pebble logs temporal-worker -f
```

Comment:
This command just hung for me. I looked at the logs with `k logs temporal-worker-k8s-0 -c temporal-worker -f` instead. Might be nice to specify what exactly in the logs I should look for, is it just the absence of errors?